### ISSUE TITLE
Should some rules ignore empty rules?

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#160]
Given an input of:

```css
.selector-ii {}
```

we get 3 warnings with the SUITCSS config:

```console
# postcss-log-warnings

css\input.css
1:1     Expected single space before "}" of a single-line block (block-closing-brace-space-before) [stylelint]
1:1     Unexpected empty block (block-no-empty) [stylelint]
1:1     Expected a trailing semicolon (rule-trailing-semicolon) [stylelint]
```

Although, the warnings are technically correct... I'm wondering whether rules like `rule-trailing-semicolon` and `block-closing-brace-space-before` should ignore empty blocks? Leaving just the `block-no-empty` warning.

Seems like an early return e.g. `if (block.nodes.length === 0) { return }` would suffice if we want to make this change.

**Comment by davidtheclark** [comment]
I think that's a good idea.
